src/components/Table/TableCell.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import cn from 'classnames';
+
+import classes from './TableCell.module.css';
+import type { SortHandler } from './Context';
+import { useTableRowTypeContext, Variant } from './Context';
+import { ReactComponent as SortIcon } from './sort_arrow.svg';
+
+export interface TableCellProps {
+  children?: React.ReactNode;
+  variant?: string;
+  colSpan?: number;
+  type?: string;
+  src?: string;
+  alt?: string;
+  sortable?: boolean;
+  onChange?: SortHandler;
+  sortDirecton?: SortDirection;
+  id?: number;
+}
+export enum SortDirection {
+  Descending = 'desc',
+  Ascending = 'asc',
+  NotSortable = 'notSortable',
+  NotActive = 'notActive',
+}
+
+export const TableCell = ({
+  children,
+  colSpan = 1,
+  variant,
+  onChange,
+  sortDirecton = SortDirection.NotSortable,
+  id,
+}: TableCellProps) => {
+  const { variantStandard } = useTableRowTypeContext();
+
+  const handleChange = () => {
+    if (onChange != undefined && id != undefined && sortDirecton != undefined) {
+      onChange({
+        idCell: id,
+        previousSortDirection: sortDirecton,
+      });
+    }
+  };
+  return (
+    <>
+      {(variant == undefined
+        ? variantStandard === Variant.Header
+        : variant === 'header') && (
+        <th
+          className={cn(classes['header-table-cell'])}
+          colSpan={colSpan}
+        >
+          <div
+            className={
+              sortDirecton != SortDirection.NotSortable
+                ? cn(classes['container-sortable'])
+                : cn(classes['container'])
+            }
+            onClick={() => handleChange()}
+            onKeyUp={() => handleChange()}
```

Comment:
Sorting: `onKeyUp` needs to check the key being pressed the same way you did for pagination, so that only `Enter` and `Space` can trigger it. Currently, any key will trigger `handleChange` as long as it is in focus, even the tab key.

Only using tab to go back and forth triggeres `handleChange`:

https://user-images.githubusercontent.com/47412359/198250379-4daf597e-b3cc-4861-9742-0494b3f825a2.mov